src/super_gradients/training/transforms/processing.py

```diff
@@ -242,3 +287,20 @@ def default_ppyoloe_coco_processing_params() -> Tuple[List[str], Processing]:
     )
     class_names = COCO_DETECTION_CLASSES_LIST
     return class_names, image_processor
+
+
+def default_deciyolo_coco_processing_params() -> Tuple[List[str], Processing]:
+    """Processing parameters commonly used for training DeciYolo on COCO dataset."""
+    from super_gradients.training.datasets.datasets_conf import COCO_DETECTION_CLASSES_LIST
```

Comment:
Is this because of circular imports ?

Reply:
It was but actually it's fine when moved outside of function, so I took it out: 3262e2ab3d32b7cbcd640330984bded8a673f1c3